src/app/api/team.json

```diff
@@ -40,7 +40,7 @@
       "name": "Thanaree Srirawewongsa (Jean)",
       "position": "Vice President of Campus Life",
       "src": "/Team/Executives/Thanaree Srirawewongsa (Jean).jpg",
-      "alt": "Thanaree Srirawewongsa (Jean)"
+      "alt": "Jean"
```

Comment:
Any reason why the alt text was changed ?

Reply:
Fixed, was running into some problems earlier with the `()`. But is revolved now, so back to normal.